pathfinder/paizo/adventure_path/ruins_of_azlant/players_guide/roapg_abilities.lst

```diff
@@ -0,0 +1,22 @@
+Employee		DESCISPI:YES	KEY:Trait ~ Employee	CATEGORY:Special Ability	TYPE:Trait.CampaignTrait	PREMULT:1,[PREABILITY:1,CATEGORY=Special Ability,Trait ~ Employee],[PREVAREQ:BypassTraitRestriction,1],[!PREABILITY:1,CATEGORY=Special Ability,TYPE.CampaignTrait]	DESC:You have worked for the Bountiful Venture Company or you are the child of one of its current employees. This connection helped you get a leg up onthe competition to get involved in the establishment of  Talmandor’s Bounty. The company didn’t want nepotism to show through too strongly, so you weren’t included in the first wave of colonists and instead have been selected to accompany the second wave of colonists to the island. You don’t want people to know that you’re part of the company, but your contacts in the Bountiful Venture Company didn’t necessarily tell you that you needed to keep it a secret. You begin play with an additional 150 gp of equipment and know (at least in some capacity) one of the other company employees: Lyra Heatherly, Perrell Beys, or Ramona Avandth. Due to your association with the Bountiful Venture Company, your words and ideas may carry additional weight with the other employees of the company. (Your GM can grant a +1 circumstance bonus on Diplomacy checks in regards to these people in certain situations.) In addition, you gain a +1 trait bonus on Bluff checks, and Bluff is always a class skill for you.			CHOOSE:STRING|Lyra Heatherly,Perrell Beys,Ramona Avandth		BONUS:SKILL|Bluff|1|TYPE=Trait	CSKILL:Bluff		SOURCEPAGE:p. 10	ASPECT:SkillBonus|+1 circumstance bonus on Diplomacy checks in regards to %LIST												
```

Comment:
The choose syntax is incorrect. You want to use this `CHOOSE:STRING|Undead|Constructs|Monkeys` not the choices are Pipe '|' separated.
Text may or may not work with the substitution. The standard is always use '%x'|Var notation method for consistent behavior. 

`ASPECT:SkillBonus|+1 circumstance bonus on Diplomacy checks in regards to %LIST`
becomes
`ASPECT:SkillBonus|+1 circumstance bonus on Diplomacy checks in regards to %1|%LIST`